addons/mail/static/src/model/model_field.js

```diff
@@ -458,7 +458,8 @@ export class ModelField {
      * @returns {any}
      */
     read(record) {
-        return record.__values.get(this.fieldName);
+		if (record.__values != undefined) {
+	        return record.__values.get(this.fieldName);}
```

Comment:
If the suggested fix in `composer_view`, can remove this diff

Reply:
thank you for review, i tried again and and bug has been fixed